### PR TITLE
Develop tutmarch antwerpen

### PR DESCRIPTION
**Pull request by klust**
Some minor layout and other corrections we made at Antwerp before our March tutorial session, rebased on the latest master at time of writing.

### CODE REVIEW

File: intro-HPC/title.tex [hunk 1]
```diff
@@ -35,7 +35,7 @@
 
 Kenneth Hoste, Toon Willems, Jens Timmerman (UGent)
 
-Geert-Jan Bex, Mag Selwa (UHasselt and KULeuven)
+Geert-Jan Bex (UHasselt and KU~Leuven)
```
Comment (boegel) [inline intro-HPC/title.tex]: Mag should be listed somewhere, right?

**Comment by boegel** [comment]
@klust the changes are *very* minimal here... :)

File: intro-HPC/title.tex (hunk 1 above)
Comment (klust) [inline intro-HPC/title.tex]: Indeed, she should have been on a separate line. I've added a commit to the branch, I don't know if it is visible without resubmitting the pull request? I do see it here, so I guess it is OK.

**Comment by boegel** [comment]
not much to hold back, going in, thanks @klust!

**Pull request merged**